code/__DEFINES/__VOID_DEFINES/overmap.dm

```diff
@@ -1,3 +1,46 @@
 #define SHUTTLE_INACTIVE_CREW 0
 #define SHUTTLE_SSD_CREW 1
 #define SHUTTLE_ACTIVE_CREW 2
+
+#define OVERMAP_GENERATOR_SOLAR "solar_system"
+#define OVERMAP_GENERATOR_RANDOM "random"
+
+//Add new star types here
+#define SMALLSTAR 1
+#define TWOSTAR 2
+#define MEDSTAR 3
+#define BIGSTAR 4
+
+//Star classes
+#define STARO 1 //Extremely bright blue main sequence star or (super)giant
+#define STARB 2 //Bright blue main sequence star or (super)giant
+#define STARA 3 //Light blue main sequence star
+#define STARF 4 //White main sequence star
+#define STARG 5 //Yellow main sequence star or supergiant
+#define STARK 6 //Orange dwarf, main sequence star, or hypergiant
+#define STARM 7 //Red dwarf or red (super)giant
+#define STARL 8 //Cool red dwarf
+#define START 9 //Methane dwarf
+#define STARY 10 //Sad lame brown dwarf
+#define STARD 11 //White dwarf
```

Comment:
```suggestion
/// Extremely bright blue main sequence star or (super)giant
#define STARO 1
/// Bright blue main sequence star or (super)giant
#define STARB 2
/// Light blue main sequence star
#define STARA 3
/// White main sequence star
#define STARF 4
/// Yellow main sequence star or supergiant
#define STARG 5
/// Orange dwarf, main sequence star, or hypergiant
#define STARK 6
/// Red dwarf or red (super)giant
#define STARM 7
/// Cool red dwarf
#define STARL 8
/// Methane dwarf
#define START 9
/// Sad lame brown dwarf
#define STARY 10
/// White dwarf
#define STARD 11
```

dissapointingly, these aren't named STAR_BLUE or other things, that would actually make sense.